rspace/src/main/scala/coop/rchain/rspace/RSpaceOps.scala

```diff
@@ -97,11 +108,105 @@ abstract class RSpaceOps[F[_], C, P, E, A, R, K](val store: IStore[C, P, A, K],
   ): F[Option[(K, Seq[R])]] = syncF.delay {
     Kamon.withSpan(installSpan.start(), finishSpan = true) {
       store.withTxn(store.createTxnWrite()) { txn =>
-        install(txn, channels, patterns, continuation)
+        install(txn, channels, patterns, continuation, true)
       }
     }
   }
 
+  private[this] def install(
```

Comment:
This function looks overly complicated. An install should bypass looking for matching continuations and write directly to the store.